### DISCUSSION TITLE
Get info about rate-limit configuration used

### DISCUSSION THREAD

**Discussion by mrVragec** [MarcGiffing/bucket4j-spring-boot-starter#300] [Q&A]
Hello guys, 

I have a question about the rate-limit information used in the rate-limiting requests. 
So for a start, we implemented the rate-limiting and it's working, we have 9 rate-limits defined in our project and everything looks good. We can get stats about consumed/rejected count but what we can't find out is the information about which rate-limit configuration was used to limit the requests. 

Is there any way to get this information? 
BTW: we use config via application.properties and bucket4j-spring-boot-starter v0.8.1. 

Best regards,
Simon

**Comment by MarcGiffing** [comment]
You may add a custom tag. 

not testet:
bucket4j.filters[0].metrics.tags[0].key=IDENTIFICATION
bucket4j.filters[0].metrics.tags[0].expression='MY_SUPER_FILTER'
